files/tasks/rpm-test-deps.yaml

```diff
@@ -11,9 +11,10 @@
       - buildah
       - skopeo
       - runc
+      - wget
     state: present
   become: true
 - name: Download umoci
-  get_url: url="https://github.com/opencontainers/umoci/releases/download/v0.4.6/umoci.amd64" dest=/usr/local/bin/umoci
```

Comment:
Is there issue with redirects?
I am just curious :-)

Reply:
I am not sure why this doesn't work:

```
 TASK [Download umoci] **********************************************************
        fatal: [root@3.143.246.15]: FAILED! => {"changed": false, "dest": "/usr/local/bin/umoci", "elapsed": 0, "msg": "An unknown error occurred: HTTPSConnection.__init__() got an unexpected keyword argument 'cert_file'", "url": "https://github.com/opencontainers/umoci/releases/download/v0.4.6/umoci.amd64"}
```

So I tried alternative solution..;)

Reply:
sounds like https://github.com/ansible/ansible/issues/80490 :-)